README.md

```diff
@@ -34,155 +37,253 @@ secure, and production-grade cloud infrastructure.
+A [Terraform] base module for creating a `terraform-google-composer`. Composer is a managed Apache Airflow
 
-This module implements the following terraform resources
+## Getting Started
 
-- `google_resource`
-- `google_something_else`
+Most common usage of the module just setting required arguments:
 
-and supports additional features of the following modules:
+```hcl
+module "terraform-google-composer" {
+  source = "git@github.com:mineiros-io/terraform-google-composer.git?ref=v0.0.1"
 
-<!-- markdown-link-check-disable -->
-- [mineiros-io/something/google](https://github.com/mineiros-io/terraform-google-something)
-<!-- markdown-link-check-enable -->
+  name   = "example-name"
+}
+```
 
-<!--
-These are some of our custom features:
+## Module Argument Reference
 
-- **Default Security Settings**:
-  secure by default by setting security to `true`, additional security can be added by setting some feature to `enabled`
+See [variables.tf] and [examples/] for details and use-cases.
 
-- **Standard Module Features**:
-  Cool Feature of the main resource, tags
+### Top-level Arguments
 
-- **Extended Module Features**:
-  Awesome Extended Feature of an additional related resource,
-  and another Cool Feature
+#### Main Resource Configuration
 
-- **Additional Features**:
-  a Cool Feature that is not actually a resource but a cool set up from us
+- [**`name`**](#var-name): *(**Required** `string`)*<a name="var-name"></a>
 
-- _Features not yet implemented_:
-  Standard Features missing,
-  Extended Features planned,
-  Additional Features planned
--->
+  The location or Compute Engine region for the environment.
 
-## Getting Started
+- [**`project`**](#var-project): *(**Required** `string`)*<a name="var-project"></a>
 
-Most basic usage just setting required arguments:
+  The ID of the project in which the resource belongs. If it is not provided, the provider project is used.
 
-```hcl
-module "terraform-google-composer" {
-  source = "github.com/mineiros-io/terraform-google-composer?ref=v0.1.0"
-}
-```
+- [**`region`**](#var-region): *(Optional `string`)*<a name="var-region"></a>
 
-## Module Argument Reference
+  The location or Compute Engine region for the environment.
 
-See [variables.tf] and [examples/] for details and use-cases.
+- [**`labels`**](#var-labels): *(Optional `map(string)`)*<a name="var-labels"></a>
 
-### Top-level Arguments
+  User-defined labels for this environment. The labels map can contain no more than 64 entries. Entries of the labels map are UTF8 strings that comply with the following restrictions: Label keys must be between 1 and 63 characters long and must conform to the following regular expression: [a-z]([-a-z0-9]*[a-z0-9])?. Label values must be between 0 and 63 characters long and must conform to the regular expression ([a-z]([-a-z0-9]*[a-z0-9])?)?. No more than 64 labels can be associated with a given environment. Both keys and values must be <= 128 bytes in size.
```

Comment:
please use markdown formatting and paragraphs to make it more readable (please fix for all docs) you can use github to render the preview of changes to check readability.

Reply:
Added some formatting to make them more readable.